Signing Balance Transfer on Extension, Ledger and Signer (#3060)
w3f/polkadot-wiki: docs/learn/learn-balance-transfers.md

Balance transfers are used to send a balance from one account to another account. To start
transferring a balance, we will begin by using [Polkadot-JS Apps][]. This guide assumes that you've
already [created an account](learn-account-generation.md) and have some funds that are ready to be
transferred. This guide also contains instructions on how to sign the balance transfer transactions
for accounts created on Polkadot JS Extension, Ledger devices and Parity Signer App.

> ## Polkadot-JS Apps UI

Reviewer: I believe we can refer to Polkadot-JS Apps as just "Polkadot-JS Apps" or "Apps UI"

Reviewer 2: Yeah, whichever one that keeps uniformity across the Wiki and Support pages. Started discussion in Tech Ed and Community channel.

After setting your inputs correctly, click the "Make Transfer" button and confirm. Once the transfer
is included in a block you will see a green notification in the top-right corner of your screen.

### Signing on Polkadot Extension

> If your account is injected to Polkadot JS UI through the Polkadot Extension, the balance transfer transaction needs to be signed

Reviewer: ```suggestion
If you inject your account in Polkadot-JS Apps through the Polkadot-JS Extension, you need to sign the balance transfer transaction
```

> on the popup window. You can examine the method and info dropdowns highlighted in the screenshot below and ensure they match with the original transaction submitted. Sign the transaction by entering the password for the account set for this account when it was created on the Polkadot Extension.

Reviewer: ```suggestion
on the pop-up window. You can examine the method and info dropdowns (highlighted in the screenshot below) and ensure they match with the original transaction submission. Then, sign the transaction by entering the account password.
```

![extension transfer](../assets/accounts/extenstion-transaction.png)

### Signing on Ledger device

> Assuming that you imported your Ledger account to Polkadot Extension, you will be presented with a popup window for signing the balance transfer transaction. Review the transaction details on the Ledger device and approve it to sign the transaction.

Reviewer: ```suggestion
Assuming that you have imported your Ledger account to the Polkadot-JS Extension, you will see a pop-up window for signing the balance transfer transaction. Review the transaction details on the Ledger device and approve it to sign the transaction.
```

> > If you are looking for instructions in a video, watch the tutorial on how to connect [Ledger to Polkadot JS UI](https://youtu.be/7VlTncHCGPc).

Reviewer: ```suggestion
> If you are looking for instructions in a video, watch the tutorial on how to connect [Ledger to Polkadot-JS Apps](https://youtu.be/7VlTncHCGPc).
```

![Ledger transfer](../assets/accounts/ledger-transaction.png)

> ### Signing on Parity Signer App

Reviewer: ```suggestion
### Signing Transactions with Parity Signer
```

> If you imported your Parity Signer account to Polkadot JS UI directly, you will be presented with a popup window with a QR code and a camera window as shown below.

Reviewer: ```suggestion
If you directly import your Parity Signer account to Polkadot-JS Apps, you will see a pop-up window with a QR code and a camera window, as shown below.
```

> Open QR scanner on your phone's Parity Signer app and scan the QR code presented. You may be prompted to enter your PIN to sign the transaction.

Reviewer: ```suggestion
Open the QR scanner on Parity Signer and scan the QR code. You may be prompted to enter your PIN to sign the transaction.
```

> The signed extrinsic will then be available as a QR code that needs to be shown in front of your computer's camera (in the red square window shown to the right in the screenshot below).

Reviewer: ```suggestion
The signed extrinsic will then be available as a QR code that needs to be shown in front of your computer's camera (as shown in the red square window shown to the right in the screenshot below).
```

![Companion transfer](../assets/accounts/signer-transaction.png)

> If you imported your Parity Signer account to [Parity Signer Companion](https://parity.link/signer-companion), you will be presented with a popup window with a QR code as shown below.

Reviewer: ```suggestion
If you import your Parity Signer account to [Parity Signer Companion](https://parity.link/signer-companion), you will see a pop-up window with a QR code, as shown below.
```

Reviewer 2: This changes the meaning of the original sentence, as it seems to imply that the pop-up window is shown when you import the account to Parity Signer Companion, which is not the case. There are two different ways the account could have been imported - 1. Parity Signer Companion 2. Directly to the Polkadot-JS UI. Probably I'll have to re-phrase it.
```suggestion
If your account was setup on [Parity Signer Companion](https://parity.link/signer-companion) instead of Polkadot-JS Apps, you will see a pop-up window with a QR code, as shown below for this transaction.
```

![Companion transfer](../assets/accounts/companion-transfer.png)

> Scan this QR code on the Parity Signer app on your phone and you will be presented with a QR code consisting of the signed extrinsic for this transaction. Click on the **next to signing** button in the popup window shown above. You can then show the QR code on your phone within the red square window.

Reviewer: ```suggestion
Scan the QR code using Parity Signer. You will see a QR code consisting of the signed extrinsic for this transaction. Click on the **next to signing** button in the pop-up window, as shown above. You can then show the QR code on your phone within the red square window.
```
